src/Kernel32.Shared/Kernel32.Helpers.cs

```diff
@@ -293,7 +293,7 @@ public static ArraySegment<byte> ReadFile(SafeObjectHandle hFile, int nNumberOfB
         private static unsafe bool TryGetErrorMessage(FormatMessageFlags flags, void* source, int messageId, int languageId, StringBuilder sb, IntPtr[] arguments, out string errorMsg)
         {
             errorMsg = string.Empty;
-            int result = FormatMessage(
+            var result = FormatMessage(
```

Comment:
What about we change the name from result to `characterCount`  or `bufferLength`  ?